### PR TITLE
chore: lower Node.js version requirement

### PR DESCRIPTION
**Pull request by balazsorban44**

### CODE REVIEW

File: packages/create-next-app/package.json [hunk 1]
```diff
@@ -52,6 +52,6 @@
     "validate-npm-package-name": "3.0.0"
   },
   "engines": {
-    "node": ">=18.18.2"
+    "node": ">=18.7.1"
```
Comment (styfle) [inline packages/create-next-app/package.json:55]:
```suggestion
    "node": ">=18.17.0"
```

File: packages/create-next-app/package.json [hunk 2]
```diff
@@ -52,6 +52,6 @@
     "validate-npm-package-name": "3.0.0"
   },
   "engines": {
-    "node": ">=18.18.2"
+    "node": ">=18.17.1"
```
Comment (styfle) [inline packages/create-next-app/package.json:55]:
```suggestion
    "node": ">=18.17.0"
```

**Review by huozhi — DISMISSED** [review]

**Review by styfle — DISMISSED** [review]
I think we should also update CI to ensure that tests pass against this version

**Review by cb1kenobi — APPROVED** [review]

**Comment by ijjk** [comment]
## Failing test suites 
Commit: b44fd998dfe9e4ae46b8c7f79be213669b161341

`pnpm test test/integration/create-next-app/templates-app.test.ts` 
- create-next-app --app > should create TS appDir projects with --ts
- create-next-app --app > should create JS appDir projects with --js
- create-next-app --app > should create JS appDir projects with --js --src-dir
- create-next-app --app > should create Tailwind CSS appDir projects with --tailwind

<details>
<summary>Expand output</summary>

  ● create-next-app --app › should create TS appDir projects with --ts

    expect(received).toBe(expected) // Object.is equality

    Expected: true
    Received: false

      81 |   for (const file of files) {
      82 |     try {
    > 83 |       expect(existsSync(resolve(projectRoot, file))).toBe(true)
         |                                                      ^
      84 |     } catch (err) {
      85 |       require('console').error(
      86 |         `missing expected file ${file}`,

      at projectFilesShouldExist (integration/create-next-app/lib/utils.ts:83:54)
      at shouldBeTemplateProject (integration/create-next-app/lib/utils.ts:133:3)
      at integration/create-next-app/templates-app.test.ts:84:30
      at useTempDir (lib/use-temp-dir.ts:24:5)
      at Object.<anonymous> (integration/create-next-app/templates-app.test.ts:64:5)

  ● create-next-app --app › should create JS appDir projects with --js

    expect(received).toBe(expected) // Object.is equality

    Expected: true
    Received: false

      81 |   for (const file of files) {
      82 |     try {
    > 83 |       expect(existsSync(resolve(projectRoot, file))).toBe(true)
         |                                                      ^
      84 |     } catch (err) {
      85 |       require('console').error(
      86 |         `missing expected file ${file}`,

      at projectFilesShouldExist (integration/create-next-app/lib/utils.ts:83:54)
      at shouldBeTemplateProject (integration/create-next-app/lib/utils.ts:133:3)
      at integration/create-next-app/templates-app.test.ts:114:30
      at useTempDir (lib/use-temp-dir.ts:24:5)
      at Object.<anonymous> (integration/create-next-app/templates-app.test.ts:94:5)

  ● create-next-app --app › should create JS appDir projects with --js --src-dir

    expect(received).toBe(expected) // Object.is equality

    Expected: true
    Received: false

      81 |   for (const file of files) {
      82 |     try {
    > 83 |       expect(existsSync(resolve(projectRoot, file))).toBe(true)
         |                                                      ^
      84 |     } catch (err) {
      85 |       require('console').error(
      86 |         `missing expected file ${file}`,

      at projectFilesShouldExist (integration/create-next-app/lib/utils.ts:83:54)
      at shouldBeTemplateProject (integration/create-next-app/lib/utils.ts:133:3)
      at integration/create-next-app/templates-app.test.ts:146:30
      at useTempDir (lib/use-temp-dir.ts:24:5)
      at Object.<anonymous> (integration/create-next-app/templates-app.test.ts:125:5)

  ● create-next-app --app › should create Tailwind CSS appDir projects with --tailwind

    expect(received).toBe(expected) // Object.is equality

    Expected: true
    Received: false

      81 |   for (const file of files) {
      82 |     try {
    > 83 |       expect(existsSync(resolve(projectRoot, file))).toBe(true)
         |                                                      ^
      84 |     } catch (err) {
      85 |       require('console').error(
      86 |         `missing expected file ${file}`,

      at projectFilesShouldExist (integration/create-next-app/lib/utils.ts:83:54)
      at shouldBeTemplateProject (integration/create-next-app/lib/utils.ts:133:3)
      at integration/create-next-app/templates-app.test.ts:182:30
      at useTempDir (lib/use-temp-dir.ts:24:5)
      at Object.<anonymous> (integration/create-next-app/templates-app.test.ts:162:5)

</details>


Read more about building and testing Next.js in [contributing.md](https://github.com/vercel/next.js/blob/canary/contributing.md).

**Review by ijjk — DISMISSED** [review]

**Review by huozhi — APPROVED** [review]

**Pull request merged**